nucleus/modelci/data_transfer_objects/eval_function.py

```diff
@@ -0,0 +1,41 @@
+from typing import List, Optional
+
+from pydantic import BaseModel, validator
+
+from ..constants import ThresholdComparison
+
+
+class EvaluationCriteria(BaseModel):
```

Comment:
Shouldn't the object actually be called `EvaluationCriterion`? Then we'll also pass a `List[EvaluationCriterion]` to a unit test upon creation. It's also a `List[int]` and not a `List[intS]` ;)